packages/utils/src/modules-sdk/internal-module-service-factory.ts

```diff
@@ -315,10 +315,8 @@ export function internalModuleServiceFactory<
           MedusaError.Types.NOT_FOUND,
           `${
             primaryKeys.length === 1
-              ? `"${
-                  lowerCaseFirst(model.name) + upperCaseFirst(primaryKeys[0])
-                }"`
-              : `${lowerCaseFirst(model.name)} ${primaryKeys.join(", ")}`
+              ? `"${lowerCaseFirst(model.name) + " - " + primaryKeys[0]}"`
```

Comment:
thought(non-blocking): Can we do better than this?

Ideally, we have something like:
```
currencyCode
priceListId
id
moneyAmountId
```

Reply:
Wouldn't it makes you think you need to provide { currencyCode } when we expect { code } ?